### ISSUE TITLE
Jupyter: Debug Just My Code not working

### ISSUE THREAD

**Issue by flyforward2016** [microsoft/vscode-jupyter#8506]
[Here](https://github.com/microsoft/vscode-python/discussions/18140#discussioncomment-1781216) is the original bug report that I filed to vscode - Python and i was asked to direct the bug report here.

## Environment data

-   VS Code version: 1.62.3 (Universal)
-   Jupyter Extension version (available under the Extensions sidebar): v2021.10.1101450599
-   Python Extension version (available under the Extensions sidebar): v2021.11.1422169775
-   OS (Windows | Mac | Linux distro) and version: Mac BigSur
-   Python and/or Anaconda version: python 3.8.8
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): base 
-   Jupyter server running: Local 

## Expected behaviour

This is a follow up question from ([vscode-python - #18085](https://github.com/microsoft/vscode-python/discussions/18085)). #18085 was on debugging in python file. This ask is about debugging in Jupyter Notebook. I originally filed under ([vscode - #138678](https://github.com/microsoft/vscode/issues/138678)) but was suggested to move here.  

Basically, with uncheck the option " Jupyter: Debug Just My Code", I should able to debug into the code base that is not written by me, ex: debug into Pytorch or Xgboost library. However, this is not working 1.62.3 also not on 1.63.0 - insider.

Similar thing has also been observed by other users.  Please refers to this reply ([vscode - jupyter - #8413](https://github.com/microsoft/vscode-jupyter/issues/8413#issuecomment-986654301))

Please help shine some lights on this. 

 

</p>
</details>

**Comment by IanMatthewHuff** [comment]
@flyforward2016 Yes currently debugging into libraries is not supported in notebook debugging. This issue here is the one to track for a resolution on this:
https://github.com/microsoft/vscode-jupyter/issues/8146
